CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - added tools to @cumulus/integration-tests for local integration testing
 - added end to end testing for discovering and parsing of PDRs
 - `yarn e2e` command is available for end to end testing
+- added flag `useList` to parse-pdr
```

Comment:
This should go under an 'Added' header in the 'Unreleased' section. Then, when we create a release, we'll add a header for that release.